modules/photo/test/test_precomp.hpp

```diff
@@ -8,4 +8,9 @@
 #include "opencv2/ts/ocl_test.hpp"
 #include "opencv2/photo.hpp"
 
+namespace opencv_test
+{
+using namespace cv::ccm;
```

Comment:
using namespace is very bad practice. It affects everything, even, if the header is not included directly.